pytorch_lightning/utilities/seed.py

```diff
@@ -41,7 +41,10 @@ def seed_everything(seed: Optional[int] = None) -> int:
 
     try:
         if seed is None:
-            seed = os.environ.get("PL_GLOBAL_SEED", _select_seed_randomly(min_seed_value, max_seed_value))
+            if "PL_GLOBAL_SEED" in os.environ:
+                seed = os.environ["PL_GLOBAL_SEED"]
+            else:
+                seed = _select_seed_randomly(min_seed_value, max_seed_value)
```

Comment:
```suggestion
            seed = os.environ.get("PL_GLOBAL_SEED", _select_seed_randomly(min_seed_value, max_seed_value)
```
let's use dict get and define the default value

Reply:
This is what the PR fixes, we call `_select_seed_randomly` here which causes the warning message to be printed.

Reply:
but this suggestion doe not change anything on your change, right? or you say that the default is called even it is not used?